src/style.css

```
@@ -0,0 +1,69 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  flex-grow: 1;
+  flex-basis: 300px;
+  height: 300px;
+  color: white;
+  justify-content: center;
+  align-items: center;
+}
+
+.block-1 {
+  background-color: rgb(100% 0% 0%);
```

Separate values with comas
```suggestion
  background-color: rgb(100%, 0%, 0%);
```

---

And redeploy your project, cause of this, you have problems with blocks visibility